contracts/PriceOracle/PriceOracle.sol

```diff
@@ -156,10 +150,10 @@ contract PriceOracle is Ownable2Step {
      * @notice Adds a new token config to enable the contract to provide prices for a new asset
      * @param config Token config struct that contains the info for a new asset configuration
      */
-    function addConfig(TokenConfig calldata config) external onlyOwner {
+    function addConfig(TokenConfig memory config) public onlyOwner {
```

Comment:
It may actually be easier for EngOps if this function accepted an array of configs i.e `addConfig(TokenConfig[] memory configs)` so they don't have to call execute too many transactions.

Reply:
Also can you remind me why we don't hardcode the feed configs and instead allow them to be added on the fly?  The audit called out some centralization concerns with this so I'm thinking that maybe we can have a timelock contract own this so that malicious changes cannot happen immediately.

Reply:
After the initial deployment of this contract, the ownership will be handed off to the Compound multisig actually. It was an ask from them to be able to change configs so they don't have to redeploy for new assets. I kept it as one config at a time so they can vote on individual assets being added. A timelock measure though makes sense but wouldn't the Compound governance process prevent immediate malicious changes?

Reply:
Yeah I think that’s fine as long as we correctly hand it off to Compund’s timelock contract.